apps/metadata-editor/src/app/records/records-list.component.spec.ts

```diff
@@ -23,6 +24,7 @@ export class RecordTableComponent {
   @Input() records: CatalogRecord[]
   @Input() totalHits: number
   @Output() recordSelect = new EventEmitter<CatalogRecord>()
```

Comment:
See my other comment about event names, this might deserve a clarification as well :)